src/components/Icon.tsx

```diff
@@ -261,6 +261,48 @@ export const Icons = {
       <path d="M12 2C6.486 2 2 6.486 2 12C2 17.514 6.486 22 12 22C17.514 22 22 17.514 22 12C22 6.486 17.514 2 12 2ZM12 20C7.589 20 4 16.411 4 12C4 7.589 7.589 4 12 4C16.411 4 20 7.589 20 12C20 16.411 16.411 20 12 20Z" />
     </>
   ),
+  loaderCircle: (
+    <>
+      <path
+        d="M12 22C13.1046 22 14 21.1046 14 20C14 18.8954 13.1046 18 12 18C10.8954 18 10 18.8954 10 20C10 21.1046 10.8954 22 12 22Z"
+        fill="#484848"
+      />
+      <path
+        d="M12 6C13.1046 6 14 5.10457 14 4C14 2.89543 13.1046 2 12 2C10.8954 2 10 2.89543 10 4C10 5.10457 10.8954 6 12 6Z"
+        fill="#484848"
+      />
+      <path
+        d="M6.34277 19.6572C7.44734 19.6572 8.34277 18.7618 8.34277 17.6572C8.34277 16.5527 7.44734 15.6572 6.34277 15.6572C5.2382 15.6572 4.34277 16.5527 4.34277 17.6572C4.34277 18.7618 5.2382 19.6572 6.34277 19.6572Z"
+        fill="#484848"
+      />
+      <path
+        d="M17.6572 8.34277C18.7618 8.34277 19.6572 7.44734 19.6572 6.34277C19.6572 5.2382 18.7618 4.34277 17.6572 4.34277C16.5527 4.34277 15.6572 5.2382 15.6572 6.34277C15.6572 7.44734 16.5527 8.34277 17.6572 8.34277Z"
+        fill="#484848"
+      />
+      <path
+        d="M4.00002 14.001C5.10514 14.001 6.00102 13.1051 6.00102 12C6.00102 10.8949 5.10514 9.99902 4.00002 9.99902C2.8949 9.99902 1.99902 10.8949 1.99902 12C1.99902 13.1051 2.8949 14.001 4.00002 14.001Z"
+        fill="#484848"
+      />
+      <path
+        d="M20 14C21.1046 14 22 13.1046 22 12C22 10.8954 21.1046 10 20 10C18.8954 10 18 10.8954 18 12C18 13.1046 18.8954 14 20 14Z"
+        fill="#484848"
+      />
+      <path
+        d="M6.34277 8.34424C7.44734 8.34424 8.34277 7.44881 8.34277 6.34424C8.34277 5.23967 7.44734 4.34424 6.34277 4.34424C5.2382 4.34424 4.34277 5.23967 4.34277 6.34424C4.34277 7.44881 5.2382 8.34424 6.34277 8.34424Z"
+        fill="#484848"
+      />
+      <path
+        d="M17.6572 19.6582C18.7618 19.6582 19.6572 18.7628 19.6572 17.6582C19.6572 16.5536 18.7618 15.6582 17.6572 15.6582C16.5527 15.6582 15.6572 16.5536 15.6572 17.6582C15.6572 18.7628 16.5527 19.6582 17.6572 19.6582Z"
+        fill="#484848"
+      />
+    </>
+  ),
+  circleOutline: (
+    <path
+      d="M12 2C6.486 2 2 6.486 2 12C2.001 17.515 6.487 22.001 12 22.001C17.514 22.001 22 17.515 22.001 12C22.001 6.486 17.515 2 12 2ZM12 20.001C7.59 20.001 4.001 16.412 4 12C4 7.589 7.589 4 12 4C16.412 4 20.001 7.589 20.001 12C20 16.412 16.411 20.001 12 20.001Z"
+      fill="#484848"
```

Comment:
**suggestion:** Typically we do not define the `fill` property in each path. Maybe we should remove, unless we really want it hard coded.